cartridges/app_storefront_custom2/cartridge/experience/components/commerce_assets/youtubeBanner.json

```diff
@@ -0,0 +1,51 @@
+{
+    "name": "youtubeBanner",
+    "description": "A banner for youtube videos.",
+    "group": "commerce_assets",
+    "attribute_definition_groups": [
+      {
+       "id":"image",
+       "name":"Banner Image Configuration",
+       "description":"You can define the image file, size and alt text for the banner image.",
+       "attribute_definitions":[
+        {
+            "id":"id",
+            "name":"Youtube ID/Link for the video you want to be displayed",
```

Comment:
Watch out for indentations and white spaces in json files as well

Reply:
fixed